src/App.js

```diff
@@ -27,13 +27,17 @@ import router from './routes';
 import withAnalytics from './utils/Analytics';
 import AppHeaderMain from './components/appheader.main';
 import AppFooterMain from './components/appfooter.main';
+import FacebookChat from './components/facebookchat.main';
 
 import './App.less';
 
+const Config = require('Config');
+
 // eslint-disable-next-line react/no-array-index-key
 const routeComponents = router.map(({ path, component }, key) => <Route exact path={path} component={component} key={key} />);
 
 const Root = () => [
+  <FacebookChat config={Config.facebook} />,
```

Comment:
Is this going to get pulled out to Footer?

Reply:
No this is not possible, the Facebook component has to be the first thing to be displayed

Reply:
Okay. Then lets be sure to call this out in our docs for the chatbot in the other repo.